docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -315,14 +319,10 @@ def source(self, follow_additional_fields=True):
         elif self.type == 'interview order':
             # TODO: refactor this. Too much of it is assembly-line specific code
             # move into the interview YAML or a separate module/subclass
-            content += "id: " + self.interview_label + "\n"
+            content += "id: interview_order_" + self.interview_label + "\n"
             content += "code: |\n"
             content += "  # This is a placeholder to control logic flow in this interview" + "\n"
             content += "  # It was generated from interview_generator.py as an 'interview order' type question."
-            content += "  # We list one variable from each screen here, in order to control the order of questions\n"
-            content += "  # To change the order, you can rearrange, or make conditional. If you make conditional here,\n"
-            content += "  # You must also make sure it's conditional in the attachment block, or assign each variable \n"
-            content += "  # on the optional screen to DAEmpty() \n"
```

Comment:
If I recall correctly, the second half of that doesn't apply anymore, but I think the first half would help new developers who are trying to help with the project.